src/main/java/gov/cabinetofice/gapuserservice/web/UserController.java

```diff
@@ -166,6 +166,14 @@ public ResponseEntity<List<UserEmailDto>> getUserEmailsBySubs(@RequestBody() Lis
         return ResponseEntity.ok(oneLoginUserService.getUserEmailsBySubs(subs));
     }
 
+    @PostMapping("/user-emails-from-subs")
+    @PreAuthorize("hasRole('ADMIN')")
+        public ResponseEntity<List<UserEmailDto>> getUserEmailsFromSubs(
+                @RequestBody() UserSubsRequestDto req){
```

Comment:
Im assuming it works but always seen this without the parenthesis 👀 
```suggestion
                @RequestBody UserSubsRequestDto req){
```